tests/test_solvers_fairflow.py

```diff
@@ -493,12 +493,12 @@ def test_solver_fairflow_respect_minimums_3():
     assert solver.solved
     # make sure every reviewer has at least 1 paper
     nrows, ncols = res.shape if len(res.shape) == 2 else (0, 0)
-    for rix in range(nrows):
+    for rix in range(ncols):
         reviewer_count_reviews = 0
-        for pix in range(ncols):
-            if res[rix, pix] != 0:
+        for pix in range(nrows):
+            if res[pix, rix] != 0:
                 reviewer_count_reviews += 1
-        assert reviewer_count_reviews >= 1
+        assert reviewer_count_reviews >= 2
```

Comment:
why did you have to change a FairFlow test in this PR?

Reply:
@justinpayan

Reply:
I changed the assert because above, on line 560, min_papers_per_reviewer is set to 2. So it seemed like the test was actually meant to be checking if each reviewer got at least 2 papers.

Reply:
I will absolutely not mind if you don't want to merge these test suite changes, I just thought I'd suggest them.